modules/nlu/src/backend/training-pipeline.ts

```diff
@@ -643,3 +676,59 @@ class TrainingCanceledError extends Error {
     this.name = 'CancelError'
   }
 }
+function extractAugmentations(
+  intent: Readonly<{
```

Comment:
you sure have those typings somewhere, `Readonly<Intent>, Readonly<ListEntityModel>, Readonly<PatternEntity>` etc